### ISSUE TITLE
Can I ask the reason why you don't implement momentum learning?

### ISSUE THREAD

**Issue by lppllppl920** [Artelnics/OpenNN#25]
I think in fixed learning rate algorithm, it's better to have a momentum option to accelerate training.

**Comment by RoberLopez** [comment]
The gradient descent algorithm was not used very much, since the conjugate gradient, quasi-Newton method or Levenberg-Marquardt yield much better performance. 

Right now the stochastich gradient descent algorithm is being implemented, which includes momentum. 

Best regards
